Makefile

```diff
@@ -13,8 +13,8 @@ package:
 	-v ${PWD}:/go/src/github.com/CMSgov/bcda-app packaging $(version)
 
 lint:
-	docker-compose -f docker-compose.test.yml run --rm tests golangci-lint run --deadline=2m
-	docker-compose -f docker-compose.test.yml run --rm tests gosec ./...
+	docker-compose -f docker-compose.test.yml run --rm tests golangci-lint run --deadline=2m --skip-dirs=ssas
+	docker-compose -f docker-compose.test.yml run --rm tests gosec -exclude-dir=ssas ./...
 
 lint-ssas:
```

Comment:
`lint-ssas` is called by `test-ssas`, but is `test-ssas` run automatically? Or only manually? Perhaps a question for @dhgreene, @alexhova, or @DeirdreHolub

Reply:
Good question.  
Maybe we should just remove all SSAS executions from `make test` and drop them into `make test-ssas`.  
Then, we could explicitly call `make test-ssas` here (for Travis): https://github.com/CMSgov/bcda-app/blob/master/.travis.yml#L45
For Jenkins, is there an existing ticket for getting SSAS integration tests executed in the AWS pipeline?